test/test_euler.py

```diff
@@ -225,6 +225,176 @@ def inf_norm(x):
     )
 
 
+@pytest.mark.parametrize("nspecies", [0, 10])
+@pytest.mark.parametrize("dim", [1, 2, 3])
+@pytest.mark.parametrize("order", [2, 3, 4])
+def test_entropy_to_conserved_conversion(actx_factory, nspecies, dim, order):
+    """Test the inviscid rhs using a trivial constant/uniform state.
+
+    This state should yield rhs = 0 to FP.  The test is performed for 1, 2,
+    and 3 dimensions, with orders 1, 2, and 3, with and without passive species.
```

Comment:
2, 3, 4*

Reply:
Hrm, no?  This comment seems consistent with what is tested in the test where it appears.